doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -188,13 +190,6 @@ Thread
 Zigbee
 ------
 
-* Updated:
-
-  * :ref:`nrfxlib:zboss` to v3.11.4.0 and platform v5.1.5 (``v3.11.4.0+5.1.5``).
-    They contain fixes for infinite boot loop due to ZBOSS NVRAM corruption and other bugs.
-    For details, see :ref:`zboss_changelog`.
-  * :ref:`ZBOSS Network Co-processor Host <ug_zigbee_tools_ncp_host>` package to the new version v2.2.3.
-
 * Fixed an issue with Zigbee FOTA updates failing after a previous attempt was interrupted.
```

Comment:
Adding a reminder to clean up this file.

Reply:
Not cleaning up was a prevention against conflicts in case somebody modifies it on main - but I will do that early next week.